### PR TITLE
Extend close-duration for stale issues and prs

### LINKED ISSUE #323: Auto expiry of issues/PR using github bot is happening too fast
This project is going at a pace for which expiring issues/PR after 7 days is too fast.

We are moving slower than that, so we should allow more grace time for those PRs and issues.

If someone can move it to 3 weeks instead of 1 that would be lovely.

### PR DESCRIPTION
**Pull request by ckotzbauer**
Closes #323 

Signed-off-by: Christian Kotzbauer <christian.kotzbauer@gmail.com>

### CODE REVIEW

File: .github/workflows/periodics-daily.yaml [hunk 1]
```diff
@@ -19,6 +19,7 @@ jobs:
         stale-issue-label: 'no-issue-activity'
         stale-pr-label: 'no-pr-activity'
         exempt-issue-labels: 'good-first-issue'
+        days-before-close: 21
```
Comment (evrardjp) [inline .github/workflows/periodics-daily.yaml:22]: Can you also update L12 to match, thank you in advance!

**Review by evrardjp — CHANGES_REQUESTED** [review]
Small nit, else it's good!

File: .github/workflows/periodics-daily.yaml (hunk 1 above)
Comment (ckotzbauer) [reply .github/workflows/periodics-daily.yaml:22]: Thanks, good catch!

**Review by evrardjp — APPROVED** [review]
LGTM

**Pull request merged**